pennylane/transforms/__init__.py

```diff
@@ -136,4 +137,5 @@
     expand_multipar,
     expand_nonunitary_gen,
     create_expand_fn,
+    create_custom_decomp_expand_fn,
```

Comment:
This is relatively minor, but I have a preference for shorter function names 🤔 Could we get away with just `create_decomp_fn`?

Reply:
Compromise: `create_decomp_expand_fn`? I think `expand` needs to be in there for context.

Reply:
what about `create_decomp_expand` or `decomp_expand_fn`? 😆